src/components/VaultCard.jsx

```diff
@@ -0,0 +1,87 @@
+import React, {Component} from 'react'
+import axios from "axios";
+import {Link} from "react-router-dom";
+
+import {socket} from "../socket/api";
+
+
+
+export class VaultCard extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            vaults: [],
+            id:localStorage.getItem('id')
+        };
+    }
+
+    componentDidMount() {
+
+        axios.get(`http://0.0.0.0:5000/user/${this.state.id}/`,
```

Comment:
Duplicate call with `IndexPage`. If you don't use `redux` you can pass it through `props` or `context`